tsl/src/nodes/gapfill/exec.c

```diff
@@ -374,6 +374,99 @@ get_boundary_expr_value(GapFillState *state, GapFillBoundary boundary, Expr *exp
 	return gapfill_datum_get_internal(arg_value, state->gapfill_typid);
 }
 
+typedef struct collect_boundary_context
```

Comment:
Nit: our code style says types/structs should use camelcase like PostgreSQL, i.e., `CollectBoundaryContext`